storage-stack-automation/README.md

```diff
@@ -0,0 +1,36 @@
+# Automation using Lambda to Deploy FSS- Storage Stack 
+
+Automated process to deploy FSS Storage stack on each new S3 bucket using Lambda. The storage stack will be linked to the scanner stack previously defined.
+
+![](architecture.png)
+# Deploy via CloudFormation
+
+   * If not already present, [deploy a Scanner Stack](https://cloudone.trendmicro.com/docs/file-storage-security/stack-add/#AddScanner) in your Cloud One Account
+   * Obtain the Scanner stack's ID and SQS URL
+      - The Stack ID and SQSURL can be obtained with [list stacks api call](https://cloudone.trendmicro.com/docs/file-storage-security/api-reference/tag/Stack#operation/listStacks)
+      - See [FSS API Documentation](https://cloudone.trendmicro.com/docs/file-storage-security/api-create-stack/) for details.
+   * In AWS Console > Services > CloudFormation
```

Comment:
* Go to AWS Console > Services > CloudFormation

Reply:
fixed